development/secrets-rotator/terraform/environments/prod/variables.tf

```
@@ -3,6 +3,11 @@ variable "project_id" {
   description = "Google Cloud project ID to deploy the secret rotator application to."
 }
 
+variable "project_workloads_id" {
+  type        = string
+  description = "Google Cloud project ID to add the IAM access to."
```

```suggestion
  description = "Additional Google Cloud project ID to grant the IAM permissions to keys rotator service account."
```
Added more context and details to the description so it's clear the difference from `project` variable. Similar change should be applied to other descriptions.

---

Done